addons/mail/static/src/messaging/entity/partner/partner.js

```diff
@@ -222,9 +225,10 @@ function PartnerFactory({ Entity }) {
     Partner.entityName = 'Partner';
 
     Partner.fields = {
-        directPartnerThread: one2one('Thread', {
-            inverse: 'directPartner',
+        correspondentThread: one2one('Thread', {
```

Comment:
Was already like that but `one2one` seems wrong, a partner could be a correspondent in multiple thread in theory

Reply:
Indeed, and this is especially true with livechat and DM with same correspondent.